docs/resources/event_gateway_produce_policy_encrypt.md

```diff
@@ -15,37 +15,31 @@ EventGatewayProducePolicyEncrypt Resource
 ```terraform
 resource "konnect_event_gateway_produce_policy_encrypt" "my_eventgatewayproducepolicyencrypt" {
   provider = konnect-beta
-condition = "context.topic.name.endsWith('my_suffix')"
-config = {
-encrypt = [
```

Comment:
This, along with `docs/resources/event_gateway_consume_policy_decrypt.md` is a breaking change - I've mentioned in CHANGELOG.md